plugins/ecs-bridge/engine/engine.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
IMHO this particular type name (`Engine`) is severely overloaded and broadly used in a few of our packages and does lack some clarity and precision on the type's purpose.